docs/source/ide-integration.rst

```diff
@@ -0,0 +1,139 @@
+.. _ide-integration:
+
+IDE Integration
+===============================
+
+IntelliJ
+--------
+
+Download and install the latest IntelliJ IDEA Community Edition: https://www.jetbrains.com/idea/download/
+
+Change the IntelliJ maximum memory to 2 GB or more `(instructions) <https://www.jetbrains.com/help/idea/increasing-memory-heap.html#d1366197e127>`__.
+
+For Intellij to pickup the required Tornado dependencies from the poms, go to **View > Tool Windows > Maven** and select **jdk-8, ptx-backend,
+opencl-backend** under profiles.
+
+Required Plugins:
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open IntelliJ and go to **Preferences > Plugins > Browse Repositories**.
```

Comment:
I think those instructions need to be updated

Reply:
Yes. let's do that in a separate PR.